modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+[id="run-plug-in-locally"]
+== Run your plug-in locally
+
+.Prerequisites
+* You must have the OpenShift CLI link:https://mirror.openshift.com/pub/[(`oc`)] installed.
+* You must have an OpenShift cluster running.
+* You must have link:https://www.docker.com/[Docker] or link:https://podman.io/[Podman] installed.
+
+.Procedure
+
+. In one terminal window, run:
++
+
+[source,terminal]
+----
+yarn install
+----
+
+. Run:
++
+
+[source,terminal]
+----
+yarn run start
+----
+
+. In the OpenShift CLI, run:
++
+
+[source,terminal]
+----
+oc login
+----
+
+. Run:
++
+
+[source,terminal]
+----
+yarn run start-console
+----
+
+. Visit link:http://localhost:9000/example[local host] to view the running plug-in.
+
+[id="run-plug-in-docker-and-VSCode-remote-container"]
+== Run your plug-in with Docker and VSCode Remote Container
```

Comment:
I would probably leave this out for simplicity and only cover the first way to run the plugin.